docs/content/guides/developer/app-examples/tic-tac-toe.mdx

```diff
@@ -100,22 +100,22 @@ public entry fun send_mark_to_game(
 public entry fun place_mark(game: &mut TicTacToe, mark: Mark, ctx: &mut TxContext) {
     // If we are placing the mark at the wrong turn, or if game has ended,
     // destroy the mark.
-    let addr = get_cur_turn_address(game);
+    let addr = game.get_cur_turn_address();
     if (game.game_status != IN_PROGRESS || &addr != &mark.player) {
-        delete_mark(mark);
+        mark.delete();
         return
     };
     let cell = get_cell_mut_ref(game, mark.row, mark.col);
```

Comment:
What's the feeling on using receiver functions for stuff like this?

Reply:
IMO this makes the code clearer, but @tnowacki has expressed dislike

Reply:
Fascinating. Why did I dislike it?